scm/git.go

```diff
@@ -56,6 +56,15 @@ type (
 		Avatar string
 	}
 
+	// Pipeline Execution details
+	Execution struct {
+		ID      string
+		Status  string
```

Comment:
`Status` should be normalized as an enum, otherwise every vendor probably has a different string for every status